bee-common/bee-packable/src/packable/mod.rs

```diff
@@ -65,4 +66,10 @@ pub trait Packable: Sized {
 
     /// Unpack this value from the given `Unpacker`.
```

Comment:
```suggestion
    /// Unpacks this value from the given `Unpacker`.
```
Also on the pack method please (I can't suggest the change on GH)

Reply:
Every time!